route_info_zos_test.go

```diff
@@ -0,0 +1,14 @@
+package sockaddr
+
+import "testing"
+
+func Test_zosProcessOnetstatOutput(t *testing.T) {
+	dummyOutput := "Default            127.0.0.1:9999   ABC      0000000000 TCPIPLINK"
```

Comment:
Is there only one line of output from `/bin/onetstat -r`, or should more data be included in `dummyOutput`?

```
const dummyOutput = `Default            127.0.0.1:9999   ABC      0000000000 TCPIPLINK
`
```